### ISSUE TITLE
Fix deprecation warning in specs

### ISSUE THREAD

**Issue by bertocq** [AyuntamientoMadrid/consul#1120]
Found this one:

```
Using `stub` from rspec-mocks' old `:should` syntax without explicitly enabling the syntax is deprecated. Use the new `:expect` syntax or explicitly enable `:should` instead. Called from /home/travis/build/AyuntamientoMadrid/consul/spec/shared/models/document_validations.rb:36:in `block (2 levels) in <top (required)>'.

```

**Comment by bertocq** [comment]
Another one:
```
Using `any_instance` from rspec-mocks' old `:should` syntax without explicitly enabling the syntax is deprecated. Use the new `:expect` syntax or explicitly enable `:should` instead. Called from /home/travis/build/AyuntamientoMadrid/consul/spec/lib/census_caller_spec.rb:26:in `block (3 levels) in <top (required)>'.

```

**Comment by aitbw** [comment]
In case anybody is wondering about the status of this issue: these warnings are already fixed in #1362 :smile: :tada:

**Comment by decabeza** [comment]
Closed by #1362 🎉
